### PR TITLE
Update Getting Started to help.osf.io [OSF-6062]

### PR DESCRIPTION
**Pull request by saradbowman**
## Purpose

This ticket updates links on osf.io and in user emails to reflect new documentation. Old documentation lives at osf.io/getting-started and new documentation lives at help.osf.io

## Changes

- Change links from osf.io/getting started to help.osf.io. 
- Update language from "Getting Started pages" to "Help Guides"

## Side effects

- help.osf.io uses host mapping. We cannot presently host an SSL cert for that site, so pages are http://help.osf.io. Can this cause an issue?
- Legacy links to osf.io/getting-started will not resolve to the new docs. Forwarding is needed
- There are a few remaining items on the ticket related to redirecting links. 

## Ticket

https://openscience.atlassian.net/browse/OSF-6062

### CODE REVIEW

File: website/templates/public/pages/support.mako [hunk 1]
```diff
@@ -35,13 +35,13 @@
...
+                        <h4 class="f-w-lg"><a href="http://help.osf.io" target="_blank" rel="noreferrer">OSF Guides</a></h4>
                     </div>
                     <div class="support-col-body clearfix">
-                        <p> Learn how to use the OSF for improving your research workflow. <a href="/getting-started">
-                            Getting Started </a> has step-by-step video tutorials and screenshots that show you the basics
+                        <p> Learn how to use the OSF for improving your research workflow. Read our <a href="http://help.osf.io" target="_blank" rel="noreferrer">
+                            Guides </a> for step-by-step screenshots that show you the basics
                             of project structures, version control, privacy, files, add-on support, and more! </p>
-                            <a href="/getting-started" class="btn btn-info m-t-lg pull-right" > Visit Getting Started <i class="fa fa-angle-right"></i> </a>
+                            <a href="http://help.osf.io" class="btn btn-info m-t-lg pull-right" > Visit Guides <i class="fa fa-angle-right"></i> </a>
```
Comment (brianjgeiger) [inline website/templates/public/pages/support.mako]: This one isn't opening in a new tab.

File: website/templates/emails/new_public_project.html.mako [hunk 2]
```diff
@@ -17,7 +17,7 @@
             !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
         </script>
         <br>
-        If you would like to learn more about how to take advantage of any of these features, visit our <a href="${osf_url}getting-started/#start">Getting Started page</a> or <a href="mailto:support@osf.io">drop us a line</a>.
+        If you would like to learn more about how to take advantage of any of these features, visit our <a href="http://help.osf.io" target="_blank" rel="noreferrer">Guides</a> or <a href="mailto:support@osf.io">drop us a line</a>.
```
Comment (brianjgeiger) [inline website/templates/emails/new_public_project.html.mako]: You shouldn't need to set the target or noreferrer for email links; the email client or website should take care of that for you.

File: website/templates/emails/welcome_osf4m.html.mako [hunk 3]
```diff
@@ -17,7 +17,7 @@
             <li>You have one place to reference when looking for your research materials</li>
             <li>You can monitor interest in your data and materials by tracking downloads, just like you can for your ${conference} presentation.</li>
         </ul>
-        To learn more about how the OSF can help you manage your research, begin with these (short!) videos on our <a href="${osf_url}getting-started">Getting Started</a> page. Or, read about how others use the OSF from a <a href="https://osf.io/7a8gs/">case study</a>.
+        To learn more about how the OSF can help you manage your research, read our <a href="http://help.osf.io" target="_blank">Guides</a>. Or, read about how others use the OSF from a <a href="https://osf.io/svje2">case study</a>.
```
Comment (brianjgeiger) [inline website/templates/emails/welcome_osf4m.html.mako]: No need for target here

**Comment by brianjgeiger** [comment]
Just a few small changes and this is good. 🐧

**Comment by sloria** [comment]
I'll fix this up and merge it.

**Comment by saradbowman** [comment]
I just fixed it!

**Comment by sloria** [comment]
I've made these changes in https://github.com/CenterForOpenScience/osf.io/commit/0a89f4ea664668f534caa005c9b19c9d375adb0a, as well as updated the OSF4I welcome email and set up redirects for http://osf.io/getting-started

**Pull request closed without merging**